src/components/safe-messages/MsgList/index.tsx

```diff
@@ -0,0 +1,17 @@
+import type { ReactElement } from 'react'
+import type { SafeMessageListPage } from '@safe-global/safe-gateway-typescript-sdk'
+
+import { TxListGrid } from '@/components/transactions/TxList'
+import MsgListItem from '@/components/safe-messages/MsgListItem'
+
+const MsgList = ({ items }: { items: SafeMessageListPage['results'] }): ReactElement => {
+  return (
+    <TxListGrid>
+      {items.map((item, i) => (
+        <MsgListItem item={item} key={i} />
```

Comment:
Do they have an id or something else to use for the key?

Reply:
The array is made up of messages and date labels. We could use something like `isSafeMessageListItem(item) ? item.messageHash : item.timestamp` but I don't think it's worth it.